packages/react-core/src/components/Badge/Badge.tsx

```diff
@@ -26,6 +29,11 @@ export const Badge: React.FunctionComponent<BadgeProps> = ({
   >
     {children}
     {screenReaderText && <span className="pf-v5-screen-reader">{screenReaderText}</span>}
+    {isToggle && (
+      <span className="pf-v5-c-badge__toggle-icon">
+        <i className="fas fa-caret-down"></i>
```

Comment:
I reckon you'll want to import a react-icon for this. 

```
import CaretDownIcon from '@patternfly/react-icons/dist/esm/icons/caret-down-icon'; // at the top with the imports
```

And then update this to:
```suggestion
        <CaretDownIcon />
```

Reply:
Ah yes, much cleaner, ty... done!